src/index.ts

```diff
@@ -588,16 +603,39 @@ export class Bigtable {
     );
   }
 
+  getInstances(callback: GetInstancesCallback): void;
+  getInstances(gaxOptions: CallOptions, callback: GetInstancesCallback): void;
+  getInstances(gaxOptions?: CallOptions): Promise<GetInstancesResponse>;
+  /**
+   * Query object for listing instances.
+   *
+   * @typedef {object} CallOptions
```

Comment:
I would advocate that we do not include a JSDoc typedef for this, just because if this is likely to become stale and overlooked. If anything lets use either an [`@external` tag](https://jsdoc.app/tags-external.html) or just link to the upstream docs.